README.md

```diff
@@ -86,6 +86,22 @@ consumer.prometheus.metrics.reporter.listener=http://:8081
 consumer.auto.include.jmx.reporter=false
 ```
 
+### MirrorMaker
+
+The Source and Checkpoints MirrorMaker connectors expose [metrics](https://kafka.apache.org/documentation/#georeplication-monitoring). To collect these metrics with the reporter, add the metrics-reporter configurations to the connector configurations. Use the same listener (`prometheus.metrics.reporter.listener`) as Kafka Connect so that all metrics are exposed together on same endpoint.
```

Comment:
```suggestion
The Source and Checkpoint MirrorMaker connectors expose [metrics](https://kafka.apache.org/documentation/#georeplication-monitoring). To collect these metrics with the reporter, add the metrics-reporter configurations to the connector configurations. Use the same listener (`prometheus.metrics.reporter.listener`) as Kafka Connect so that all metrics are exposed together on same endpoint.
```